irohad/network/impl/block_loader_impl.hpp

```diff
@@ -35,12 +36,16 @@ namespace iroha {
       BlockLoaderImpl(
           std::shared_ptr<ametsuchi::PeerQuery> peer_query,
           std::shared_ptr<ametsuchi::BlockQuery> block_query,
-          std::shared_ptr<model::ModelCryptoProvider> crypto_provider);
+          std::shared_ptr<model::ModelCryptoProvider> crypto_provider,
+          std::shared_ptr<shared_model::validation::BlockValidator> =
```

Comment:
BlockValidator is designed to be a template, but now it's a stub, so you should not instantiate it, as later it can change, use DefaultBlockValidator(=BlockValidator). Or should we use a template here?